microsoft-edge/webview2/release-notes.md

```diff
@@ -104,6 +104,78 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
 <!-- end of top matter -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2045.28
+
+Release Date: September 18, 2023<!-- todo: finalize -->
+
+[NuGet package for WebView2 SDK 1.0.2045.28](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2045.28)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 117.0.2045.28 or higher.
+
+#### General

```

Comment:
```suggestion
#### General

###### Promotions

No additional APIs have been promoted to Stable and added in this Release SDK.

```
add this heading and statement like in August Release?  @victorhuangwq @peiche-jessica

Reply:
pushed a commit impl'g that